include/random_walks/boundary_cdhr_walk.hpp

```diff
@@ -9,7 +9,7 @@
 #define RANDOM_WALKS_BOUNDARY_CDHR_WALK_HPP
 
 #include "sampling/sphere.hpp"
-
+#include<cmath>
```

Comment:
please follow the coding style of the rest of the library i.e. write `#include <cmath>`